File: Iniciante/1073 - Quadrado de Pares/Python/1073.py
def main():
    N = int(input())

    quadrado(N)


def quadrado(n):
    i = 1

    while i <= n:
        if n % 2 != 0 and i != n:
            i += 1
        if i % 2 == 0:
            print(f"{i}^2 = {i ** 2}")
        i += 1


main()

File: Iniciante/1073 - Quadrado de Pares/JavaScript/1073.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");

  const N = parseInt(input);

  quadrado(N);
}

function quadrado(n) {
  let i = 1;

  while (i <= n) {
    if (n % 2 !== 0 && i !== n) {
      i++;
    }
    if (i % 2 === 0) {
      console.log(`${i}^2 = ${i ** 2}`);
    }
    i++;
  }
}

main();
